main.py

```diff
@@ -88,25 +88,25 @@ def _kill_tb_child():
 def get_best_checkpoint_path(run_dir):
     """ Look in run_dir for model checkpoint to load.
     Hierarchy is
-        1) best checkpoint from eval (target_task_training)
+        1) best checkpoint from target_task_training
         2) best checkpoint from pretraining
         3) checkpoint created from before any target task training
         4) nothing found (empty string) """
-    eval_best = glob.glob(os.path.join(run_dir, "model_state_eval_best.th"))
-    if len(eval_best) > 0:
-        assert_for_log(len(eval_best) == 1,
+    target_task_best = glob.glob(os.path.join(run_dir, "model_state_target_train_best.th"))
```

Comment:
Send out some @channel messages about this once it's checked in, since this breaks old checkpoints.

Reply:
Or try to set this up so it'll recognize old checkpoints.